test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -1,9 +1,16 @@
 require "integration_test_helper"
 require "gds_api/test_helpers/mapit"
+require_relative "../support/coronavirus_local_restrictions_helpers"
 
 class CoronavirusLocalRestrictionsTest < ActionDispatch::IntegrationTest
```

Comment:
Nice one on the tests removed from here. I think you can remove more too and may want to adjust the phrasing on ones we keep to better capture their purpose.

I think it could suffice with (note: comments intended to communicate intent rather than usage in the code):

```
  it "displays restrictions for an area with restrictions" do
    given_i_am_on_the_local_restrictions_page
    # then_i_can_see_the_postcode_lookup_form # I don't think you need this assertion as entering a postcode will implicitly test it
    # and_there_is_metadata # this doesn't seem a feature concern
    # then_i_enter_a_valid_english_postcode_in_tier_one
    when_i_enter_a_english_postcode_for_a_restriction_area
    then_i_click_on_find
    # then_i_see_the_results_page_for_level_one
    then_i_see_the_restriction_information
  end

  it "displays guidance for a devolved nation" do
    given_i_am_on_the_local_restrictions_page
    # then_i_can_see_the_postcode_lookup_form # tested implicitly
    # then_i_enter_a_valid_welsh_postcode
    when_i_enter_a_postcode_for_a_devolved_nation_area
    then_i_click_on_find
    # then_i_see_the_results_for_wales
    then_i_see_the_devolved_nation_guidance
  end

  it "displays no information for an area without tier information" do
    given_i_am_on_the_local_restrictions_page
    # then_i_can_see_the_postcode_lookup_form # tested implicitly
    then_i_enter_a_valid_english_postcode_with_no_tier_information
    then_i_click_on_find
    then_i_see_the_no_information_page
  end

  describe "with javascript" do
    before { Capybara.current_driver = Capybara.javascript_driver }
    after { Capybara.use_default_driver }

    it "normalises the submitted postcode" do
      given_i_am_on_the_local_restrictions_page
      then_i_can_see_the_postcode_lookup_form
      then_i_enter_an_unusually_formatted_postcode
      then_i_click_on_find
      then_i_see_the_results_with_conventional_postcode_formatting
    end
  end
```

And then all the other feature tests here deleted.

Reply:
Thanks, will do this Monday before stand up!